pkg/services/tree/service.go

```diff
@@ -122,7 +123,7 @@ func (s *Service) Remove(_ context.Context, req *RemoveRequest) (*RemoveResponse
 	b := req.GetBody()
 	cid := getCID(b.GetContainerId())
 
-	err := s.verifyClient(req, cid, req.GetSignature().GetKey())
+	err := s.verifyClient(req, cid, b.GetBearerToken(), eacl.OperationPut)
```

Comment:
Are we going to define dedicated operations for Tree service?

Reply:
We could, but there is no necessity currently. It is more of an auxilliary service.